### PR TITLE
TRI-143/Implement resume ticket method

### PR DESCRIPTION
**Pull request by luccar31**
# Pull Request

## Description
Same as pause method, but with this once the timer can be resumed.

## Ticket Link
[TRI-143](https://linear.app/tricker-sirius/issue/TRI-143/implement-resume-ticket-method)

## Steps to Reproduce
1. Run `prisma generate`
2. Run `docker compose up`
3. Check Swagger documentation for [resume endpoint](http://localhost:8080/api-docs/#/Issue/get_api_issue__issueId__resume)
4. Test the endpoint with Postman or Swagger.

### CODE REVIEW

File: src/documentation/domains/issue/controller/issue.controller.docs.ts [hunk 1]
```diff
@@ -1,9 +1,35 @@
 /**
  * @swagger
  * paths:
+ *   /api/issue/{issueId}/resume:
```
Comment (Juan-Bianchi) [inline src/documentation/domains/issue/controller/issue.controller.docs.ts:4]: Please add security to the endpoint, it cannot be tested in swagger.

File: src/domains/issue/controller/issue.controller.ts [hunk 2]
```diff
@@ -34,6 +36,14 @@ issueRouter.get('/:issueId/pause', validateRequest(IssuePauseParams, 'params'),
   return res.status(HttpStatus.OK).json(event);
 });
```
Comment (Juan-Bianchi) [inline src/domains/issue/controller/issue.controller.ts:47]: Just a comment. I was thinking that as your are making changes in database, the resume and pause endpoint should have another verb (post, put)

**Review by Juan-Bianchi — CHANGES_REQUESTED** [review]
Endpoint is working properly with postman, but as security has not been added in docs, it cannot be tested in Swagger.

File: src/documentation/domains/issue/controller/issue.controller.docs.ts (hunk 1 above)
Comment (luccar31) [reply src/documentation/domains/issue/controller/issue.controller.docs.ts:4]: Done.

File: src/domains/issue/controller/issue.controller.ts (hunk 2 above)
Comment (luccar31) [reply src/domains/issue/controller/issue.controller.ts:47]: Yes, I was thinking about this and a I did a little research on Google, and found [this](https://stackoverflow.com/questions/611906/http-post-with-url-query-parameters-good-idea-or-not).
The SO post explains when a method should be a POST o something else. I think that my method, as I'm creating new events and thus changing server state, should be post. Same pause method, but I will do that change in the respective PR.

**Review by Juan-Bianchi — APPROVED** [review]

**Pull request merged**